src/llmq/net_signing.cpp

```
@@ -4,44 +4,130 @@
+        }
+        if (!ranges::all_of(msgs, [this, &pfrom](const auto& ann) {
+                return m_shares_manager->ProcessMessageSigSesAnn(pfrom, ann);
+            })) {
+            BanNode(pfrom.GetId());
+            return;
+        }
+    } else if (msg_type == NetMsgType::QSIGSHARESINV || msg_type == NetMsgType::QGETSIGSHARES) {
+        std::vector<CSigSharesInv> msgs;
+        vRecv >> msgs;
+        if (msgs.size() > CSigSharesManager::MAX_MSGS_CNT_QSIGSHARES) {
+            LogPrint(BCLog::LLMQ_SIGS, "NetSigning::%s -- too many invs in %s message. cnt=%d, max=%d, node=%d\n",
```

Cppcheck (2.19.0)

[error] Uninitialized variable

(legacyUninitvar)